README.md

```diff
@@ -6,6 +6,11 @@ Robert N. M. Watson (University of Cambridge),
 Konrad Witaszczyk (University of Cambridge), and
 Jessica Man (University of Cambridge)
 
+<div class="warning">
+This tutorial covers the obsolete 23.11 release.  Some material may be
+out of date.
```

Comment:
It might be worth adding:
```suggestion
This tutorial covers the obsolete 23.11 release.  Some material may be
out of date.
See the [CheriBSD features section](https://ctsrd-cheri.github.io/cheribsd-getting-started/features/) in the Getting Started with CheriBSD guide for information on features in the latest CheriBSD release.
```